packages/donpapi/PKGBUILD

```diff
@@ -4,15 +4,17 @@
 pkgname=donpapi
 _pkgname=DonPAPI
 pkgver=V1.2.0.r40.g61db37a
```

Comment:
it's not tagged v2 yet because https://github.com/login-securite/DonPAPI/issues/116